docs/rackhd/rediscovery.rst

```diff
@@ -0,0 +1,146 @@
+              "options": {
+                  "reset-at-start": {
+                      "nodeId": "<nodeId>"
+                  },
+                  "discovery-refresh-graph": {
+                      "graphOptions": {
+                          "target": "<nodeId>"
+                      },
+                      "nodeId": "<nodeId>"
+                  },
+                  "generate-sku": {
+                      "nodeId": "<nodeId>"
+                  },
+                  "generate-enclosure": {
+                      "nodeId": "<nodeId>"
+                  },
+                  "create-default-pollers": {
+                      "nodeId": "<nodeId>"
+                  },
+                  "run-sku-graph": {
+                      "nodeId": "<nodeId>"
+                  },
+                  "nodeId": "<nodeId>"
+              }
+            }' \
+        <server>/api/2.0/workflows
+
+
+Delayed Refresh Node Discovery
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+An user can defer a node discovery by posting to /api/2.0/workflows with a payload. The user will need to manually reboot the node after executing the API before the node re-discovery/refresh process can start.
+
+**Delayed Node Re-discovery example**
+
+.. code-block:: REST
+
+    POST /api/2.0/workflows
+    {
+        "name": "Graph.Refresh.Delayed.Discovery",
+        "options": {
+            "discovery-refresh-graph": {
+                "graphOptions": {
+                    "target": "<nodeId>"
+                },
+                "nodeId": "<nodeId>"
+            },
+            "generate-sku": {
+                "nodeId": "<nodeId>"
+            },
+            "generate-enclosure": {
+                "nodeId": "<nodeId>"
+            },
+            "create-default-pollers": {
+                "nodeId": "<nodeId>"
+            },
+            "run-sku-graph": {
+                "nodeId": "<nodeId>"
+            },
+            "nodeId": "<nodeId>"
+        }
+    }
+
+.. code-block:: REST
+
+   curl -X POST \
+        -H 'Content-Type: application/json' \
+        -d '{ "name":"Graph.Refresh.Delayed.Discovery",
+              "options": {
+                  "discovery-refresh-graph": {
+                      "graphOptions": {
+                          "target": "<nodeId>"
+                      },
+                      "nodeId": "<nodeId>"
+                  },
+                  "generate-sku": {
+                      "nodeId": "<nodeId>"
+                  },
+                  "generate-enclosure": {
+                      "nodeId": "<nodeId>"
+                  },
+                  "create-default-pollers": {
+                      "nodeId": "<nodeId>"
+                  },
+                  "run-sku-graph": {
+                      "nodeId": "<nodeId>"
+                  },
+                  "nodeId": "<nodeId>"
+              }
+            }' \
+        <server>/api/2.0/workflows
+
+**Manually rebooting the node using ipmitool example**
+
+.. code-block:: REST
```

Comment:
Should this example be in the form of a workflow invocation instead of a direct ipmi call?